.github/workflows/stale_issue.yml

```diff
@@ -0,0 +1,49 @@
+name: "Close stale issues"
+
+# Controls when the action will run.
+on:
+  push:
```

Comment:
The test looks good to me. So let's remove the `push` condition before we merge it into mainline.